terraform/development/main.tf

```diff
@@ -84,7 +84,8 @@ module "bulk_upload" {
 module "cds_export" {
   source = "../modules/cds_export"
 
-  prefix = local.prefix
+  prefix              = local.prefix
+  cds_access_role_arn = "arn:aws:iam::837698168072:role/developer" # TODO CLDC-2764 update role arn to that the CDS team will use

```

Comment:
Just placeholders for each env right now (terraform moans if left as an empty string)

Reply:
Rather than providing something here that we don't want to apply the policy to, can we make this optional and have the module not set up the role/policy if it isn't provided?

(Particularly since long term I expect this not to be necessary for dev and staging)

Reply:
Yeah good point! I've updated this now such that nothing will be setup for all envs as things stand, but can provide the cds role arn later for whichever env(s) needing it